### PR TITLE
[Selenium] Fix unstable selenium tests from dashboard and organizations packages

### PR DESCRIPTION
**Pull request by SkorikSergey**
### What does this PR do?
Fix next unstable selenium tests from dashboard and organizations packages:
- WorkspacesListTest
- RenameWorkspaceTest
- ShareWorkspaceOwnerTest

### What issues does this PR fix or reference?
https://github.com/eclipse/che/issues/15392

### CODE REVIEW

**Comment by SkorikSergey** [comment]
ci-test

**Comment by SkorikSergey** [comment]
ci-test

**Comment by SkorikSergey** [comment]
ci-test

**Comment by SkorikSergey** [comment]
ci-test

**Comment by SkorikSergey** [comment]
ci-test

**Comment by SkorikSergey** [comment]
ci-test

**Review by artaleks9 — APPROVED** [review]

File: tests/legacy-e2e/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/dashboard/workspaces/WorkspaceDetails.java [hunk 1]
```diff
@@ -57,8 +57,7 @@
     String APPLY_CHANGES_BUTTON = "//che-button-save-flat[@class='apply-button']";
     String CANCEL_CHANGES_BUTTON = "//button[@name='cancel-button']";
     String CANCEL_DIALOG_BUTTON = "//md-dialog[@role='dialog']//button/span[text()='Cancel']";
-    String CLOSE_DIALOG_BUTTON =
-        "//*[@id='cancel-dialog-button']//*[contains(@class, 'che-button')]";
+    String CLOSE_DIALOG_BUTTON = "close-dialog-button";
```
Comment (dmytro-ndp) [inline tests/legacy-e2e/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/dashboard/workspaces/WorkspaceDetails.java]: `CLOSE_DIALOG_BUTTON` > `CLOSE_DIALOG_BUTTON_NAME` will make it clear how this constant should be applied.

**Review by dmytro-ndp — APPROVED** [review]
Looks good.

**Pull request merged**